src/test/java/core/basesyntax/service/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service;
```

Comment:
```suggestion
package core.basesyntax.strategy;
```
Move this test to appropriate folder.